src/components/dashboard/FaceRecognition/FRIntro.js

```diff
@@ -22,7 +22,7 @@ const FRIntro = props => {
   const isValid = props.screenProps.screenState && props.screenProps.screenState.isValid
 
   log.debug({ isIOS, isMobileSafari })
-  if (isUnsupported) {
+  if (!isUnsupported) {
```

Comment:
this looks like a typo, also our linter doesn't allow negated conditions did you disable the linter?

Reply:
yes it is type. I've set it to test unsupported screen. I don't disable lint

Reply:
but I've already return this back

Reply:
it is just middle commit. before create a PR